src/main/java/com/synopsys/integration/alert/common/digest/DigestNotificationProcessor.java

```diff
@@ -23,26 +23,32 @@
  */
 package com.synopsys.integration.alert.common.digest;
 
+import java.util.Collection;
 import java.util.List;
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 import com.synopsys.integration.alert.channel.event.ChannelEvent;
-import com.synopsys.integration.alert.common.digest.filter.NotificationEventManager;
+import com.synopsys.integration.alert.common.digest.filter.NotificationFilter;
+import com.synopsys.integration.alert.common.digest.filter.NotificationToEventConverter;
 import com.synopsys.integration.alert.common.enumeration.DigestType;
 import com.synopsys.integration.alert.database.entity.NotificationContent;
 
 @Component
 public class DigestNotificationProcessor {
```

Comment:
I think we want to rename this to just NotificationProcessor now.  Digest is a particular type of format and formatting will be separated now from the filtering of the notification data.